app/views/groups/index.html.erb

```diff
@@ -0,0 +1,58 @@
+<nav class="navbar navbar-collapse navbar-dark bg-dark">
+  <div class="container-fluid">
+    <div>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav ms-auto">
+          <li class="nav-item">
+            <%= link_to "Home", root_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "About", root_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Contact", root_path, class: "nav-link" %>
```

Comment:
- [ ] Dear @HunterX-7, great job I personally like your design 💯 👌🏻  However it's one of the project requiement we follow the provided design (width, font, size, height, background, color...)  Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [ ] Kindly follow the provided design:
 -  You need to change your `ADD CATEGORY ` height, color, and background color, ...  and add this white border around the button text. 👍🏻 
 - The NEW TRANSACTION button has a similar problem. Kindly fix it as well. 💯  👍🏻

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/236525931-a708eaab-9a2f-4e3f-b8da-46b8e4624ea2.png) | ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)

Reply:
Thank you a lot for the feedback, the suggested changes have been added! 😄
 
![image](https://user-images.githubusercontent.com/90111138/236555703-ea50d323-d846-49cb-b44a-723920ecf585.png)